tests/test_utils/test_testcase.py

```diff
@@ -38,3 +44,13 @@ def test_disallowed_database_queries(self):
         )
         with self.assertRaisesMessage(AssertionError, message):
             Car.objects.using('other').get()
+
+    def test_setup_test_data_deepcopyable(self):
```

Comment:
I feel like this should be its own test case since it's othrewise a test at quite a distance